packages/ia-topnav/src/dropdown-menu.js

```diff
@@ -55,9 +56,11 @@ class DropdownMenu extends TrackedElement {
     const calloutText = this.config.callouts?.[link.title];
     return html`<a
       href="${formatUrl(link.url, this.baseHost)}"
+      class="${link.class}"
```

Comment:
class prop always available?

Reply:
yes, ifDefined() lib giving errors in offshoot build so keep class props always.